tests/specs/spark-session.dbtspec

```diff
@@ -0,0 +1,16 @@
+target:
+  type: spark
+  method: session
+  host: localhost
+  schema: "analytics_{{ var('_dbt_random_suffix') }}"
+sequences:
+  test_dbt_empty: empty
+  # Disabled tests requires hive support
```

Comment:
I don't know yet how to enable `hive` support, this is particular hard due to the sub-process calls to `dbt` inside the `pytest-dbt-adapter` package

Reply:
Given how difficult the `.dbtspec` tests are to debug, and the fact that we're moving away from this framework, I'd be happy with an alternative integration test, to validate some truly basic functionality. The setup could be similar to the ones we have in [`tests/integration`](https://github.com/dbt-labs/dbt-spark/tree/main/tests/integration) currently. I think it would first require adding a new profile marker to the list in `conftest.py`.

Reply:
most of the test are disabled due to missing hive support, which is not easy to add due to the subprocesses in `.dbtspec`. @jtcohen6 : do you consider the two active tests to be sufficient?